vignettes/adrs_gcig.Rmd

```diff
@@ -0,0 +1,799 @@
+
+4. **Progression**: 
+
+- category A: **CA-125 increased at baseline returning to normal after treatment**: CA-125 >= 2 * ULRR on 2 occasions at least 1 week apart.
+- category B: **CA-125 increased at baseline not returning to normal after treatment**: CA-125 >= 2 * nadir value (lowest level reached by CA-125) on 2 occasions at least 1 week apart.
+- category C: **CA-125 within reference range at baseline**: CA-125 >= 2 * ULRR on 2 occasions at least 1 week apart.
+
+5. **Not Evaluable**: This is when the patient's response cannot be evaluated due to various reasons such as receiving mouse antibodies or having medical/surgical interference with their peritoneum or pleura during the previous 28 days.
+
+## Ways the data is collected in the Ovarian Cancer study CRF
+
+### Required:
+
+- Serum CA-125 results as per protocol and mapped in `LB` SDTM domain.
+- RECIST1.1 tumor size and response as per protocol and mapped in `RS`/`TR`/`TU` SDTM domain.
+
+**Note**: collection of CA-125 and RECIST 1.1 tumor assessment may not be always same visit. CA-125 can be collected more frequently than tumor assessment.
+
+### May be Collected:
+
+- CA-125 progression
+- CA-125 response
+- Combined response of CA-125 and RECIST 1.1 based on GCIG criteria
+
+### Assumptions:
+
+1.	CA-125 confirmed responses are collected on the CRF and mapped to `SDTM RS` domain. Please note that we are not programmatically confirming the CA-125 response.
+2.	The date of CA-125 response is assigned to the date of the first measurement when the CA-125 level is reduced by 50%. The date of CA-125 progression is assigned to the date of the first measurement that meets progression criteria. This is assumed to be collected on the CRF CA-125 response page.
+3.	CA-125 responses are mapped to `CR`, `PR`, `SD`, `PD`, `NE` as shown below.
+    ```{r echo=FALSE}
+list_resp <- tibble::tribble(
+  ~"CA-125 response per GCIG", 
+  ~"CA-125 response mapped",
+  "Response within Normal Range", "CR",
+  "Response but not within Normal Range", "PR",
+  "Non-Response/Non-PD", "SD",
+  "PD", "PD",
+  "NE", "NE"
+)
+
+knitr::kable(list_resp)
+    ```
+
+4.	Combined CA-125 and RECIST 1.1 response per visit where both RECIST 1.1 and CA-125 is evaluated is also collected on the CRF. The investigator should already consider CA-125 confirmed response and each visit RECIST 1.1 response based on GCIG document.
+5. In `SUPPRS` there are records with below `QNAM` and `QLABEL` values:
+
+    ```{r echo=FALSE}
+list_supp <- tibble::tribble(
+  ~"QNAM", ~"QLABEL", ~"Purpose", 
+  "CA125EFL", "CA-125 response evaluable", "CA-125 > 2x ULRR and no mouse antibodies at baseline",
+  "CA2XULRR", "CA-125 pre-treatment 2x ULRR", "CA-125 level at baseline (in range/elevated)",
+  "MOUSEANT", "Received mouse antibodies", "Prohibited therapy",
+  "CA50RED", ">50% reduction from baseline", "Response indicator",
```

Comment:
@starosto @vinhn23 CA2XULRR is used for PD A,B condition for elevated ca125 baseline. The variable name suggests at baseline it should be >=2 ULRR. I think for PD we only need to say its elevated not necessarily 2ULRR. it can be above the normal range? do you agree, if yes can we change the QNAM, QLABEL accordingly

Reply:
I agree, @vinhn23 please lmk once QNAM and QLABEL are updated

Reply:
@starosto  I updated for Stefan's comment and updated the QNAM/QLABEL to CAELEPRE/Elevated pre-treatment CA-125.

Reply:
@vinhn23 I still can not see the change here https://github.com/pharmaverse/pharmaversesdtm/blob/123_rs_ca125/data-raw/rs_supprs_onco_ca125.R